lib/client/browser.js

```diff
@@ -210,6 +210,15 @@ require("./discovery.js");
 						QUnit.begin(function(args) {
 							totalNumberOfTest += args.totalTests;
```

Comment:
`totalNumberOfTest` should also be increased by 1 before calling `karma.info`
(Similar to the QUnit error reporting above (currently line 194)).